src/Memento.jl

```diff
@@ -9,6 +9,7 @@ using Compat.Dates
 using Compat.Sockets
 using Compat.Distributed
 
+import Compat: @__MODULE__
```

Comment:
This should be unnecessary as long as there's a `using Compat`, since Compat exports it.